doc/select.md

```diff
@@ -0,0 +1,303 @@
+
+**Example:**
+
+```lua
+crud.select('developers', {{'>=', 'age', 30}, {'==', 'surname', "Adams"}})
+---
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucked_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'surname', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [3, 9661, 'Pavel', 'Adams', 27]
+...
+```
+
+## Pagination
+
+[See more](https://github.com/tarantool/crud#select) about ``opts`` parameter.
+
+### ``first`` parameter
+
+Using the ``first`` option we will get the first **N** results for the query.
+
+**Example:**
+
+```lua
+res, err = crud.select('developers', nil, { first = 3 })
+res
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucked_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'surname', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [1, 7331, 'Alexey', 'Adams', 20]
+  - [2, 899, 'Sergey', 'Allred', 21]
+  - [3, 9661, 'Pavel', 'Adams', 27]
+...
+```
+
+Thus, we got the first three objects from the ``developers`` space.
+
+### ``after`` parameter
+
+Using ``after``, we can get the objects after specified tuple.
+
+**Example:**
+
+```lua
+res, err = crud.select('developers', nil, { after = res.rows[3] })
+res
+---
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucked_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'surname', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [4, 501, 'Mikhail', 'Liston', 51]
+  - [5, 1993, 'Dmitry', 'Jacobi', 16]
+  - [6, 8765, 'Alexey', 'Sidorov', 31]
+...
+```
+
+With this request, we got objects behind the objects from the [previous example](https://github.com/tarantool/crud/doc/select#first-parameter)
+
+### Combine ``first`` and ``after`` 
+
+To use pagination, we have to combine ``after`` and ``first`` parameters. 
+
+**Example:**
+
+```lua
+res, err = crud.select('developers', nil, { first = 3 })
+res
+--- Got first three objects
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucked_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'surname', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [1, 7331, 'Alexey', 'Adams', 20]
+  - [2, 899, 'Sergey', 'Allred', 21]
+  - [3, 9661, 'Pavel', 'Adams', 27]
+...
+res, err = crud.select('developers', nil, { after = res.rows[3], first = 3 })
+res
+--- Got the next three objects
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucked_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'surname', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [4, 501, 'Mikhail', 'Liston', 51]
+  - [5, 1993, 'Dmitry', 'Jacobi', 16]
+  - [6, 8765, 'Alexey', 'Sidorov', 31]
+...
+```
+
+### Reverse pagination
+
+Select also supports reverse pagination. To use it, pass a negative value to the ``first`` parameter.
```

Comment:
As I remember you can't use reverse pagination without "after" option. Seems you could add small note about it here.